sonar-ws/src/main/java/org/sonarqube/ws/client/projectlinks/CreateWsRequest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarqube.ws.client.projectlinks;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
+
+public class CreateWsRequest {
+  @CheckForNull
```

Comment:
ping, please remove the `@CheckForNull` annotation and put it if appropriate on the get method. You could use the Builder pattern to make sure the desired parameters are not null